lib/resty/openssl/pkey.lua

```diff
@@ -88,16 +88,18 @@ local function load_pem_der(txt, opts, funcs)
           end
           arg = { null, nil, passphrase }
         elseif opts.passphrase_cb then
-          passphrase_cb = ffi_cast("pem_password_cb", function(buf, size)
-            local p = opts.passphrase_cb()
-            local len = #p -- 1 byte for \0
-            if len > size then
-              ngx.log(ngx.WARN, "pkey:load_pem_der: passphrase truncated from ", len, " to ", size)
-              len = size
-            end
-            ffi_copy(buf, p, len)
-            return len
-          end)
+          if passphrase_cb == nil then
+            passphrase_cb = ffi_cast("pem_password_cb", function(buf, size)
```

Comment:
we can say 
password_cb = password_cb or ffi_cast(...
to avoid an indention level

Reply:
I tend to prefer a `if` for that kind of things... but I don't have a strong opinion really.. I will modify.